src/PDFBuilder.js

```diff
@@ -186,6 +186,7 @@ export default class PDFBuilder extends WebformBuilder {
   afterAttach() {
     this.on('saveComponent', (component) => {
       this.webform.postMessage({ name: 'updateElement', data: component });
+      this.webform.postMessage({ name: 'form', data: this.webform.form });
```

Comment:
This is a very drastic change to fix a such a small issue. This makes me believe that all components have this problem. I would rather determine why component layout setting is not setting correctly without having to reset the whole form object. I am sure this will cause some other regressions, so this needs to be revisited.